back/spec/factories/notifications.rb

```diff
@@ -236,16 +172,8 @@
     end
   end
 
-  factory :status_change_on_initiative_you_follow, parent: :notification, class: 'Notifications::StatusChangeOnInitiativeYouFollow' do
-    association :post, factory: :initiative
-    association :post_status, factory: :initiative_status
-    before(:create) do |notification|
-      notification.post.initiative_status_changes.create!(initiative_status: notification.post_status)
-    end
-  end
-
   factory :threshold_reached_for_admin, parent: :notification, class: 'Notifications::ThresholdReachedForAdmin' do
-    association :post, factory: :initiative
+    association :post, factory: :idea
```

Comment:
TODO: On Notification model: Post -> Idea, PostStatus -> IdeaStatus

Reply:
Done in Part 3: https://github.com/CitizenLabDotCo/citizenlab/pull/9906